rust/bundle-helper/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["Stackable GmbH <info@stackable.de>"]
+description = "Stackable Bundle Helper for  the OpenPolicyAgent"
+edition = "2021"
+license = "OSL-3.0"
+name = "stackable-opa-bundle-helper"
```

Comment:
I'm not in love with calling this "helper" tbh, would have preferred something like "builder" or "manager" or similar..